### PR TITLE
Slippy tile issue 997

### PR DESCRIPTION
**Pull request by gdey**

### CODE REVIEW

**Review by ARolek — APPROVED** [review]
Lots of great clean up. Thanks for tackling this!

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/go-spatial/tegola
 
-go 1.21
```
Comment (ARolek) [inline go.mod:3]: Note, if you rebase against master this should have already been taken care of. Also we use semantic versioning now, so 1.22.5

File: provider/hana/util_internal_test.go [hunk 2]
```diff
@@ -79,7 +79,7 @@ func TestReplaceTokens(t *testing.T) {
 			sql:       "SELECT id, !pixel_width! as width, !pixel_height! as height, !scale_denominator! as scale_denom FROM foo WHERE !BBOX!",
 			layer:     Layer{srid: tegola.WebMercator, geomField: "geom"},
 			tile:      provider.NewTile(11, 1070, 676, 64, tegola.WebMercator),
-			expected:  `SELECT id, 76.43702827 as width, 76.43702827 as height, 272989.38669477 as scale_denom FROM foo WHERE "geom".ST_IntersectsRectPlanar(NEW ST_POINT($1, $3), NEW ST_POINT($2, $3)) = 1`,
+			expected:  `SELECT id, 76.43702829 as width, 76.43702829 as height, 272989.38673277 as scale_denom FROM foo WHERE "geom".ST_IntersectsRectPlanar(NEW ST_POINT($1, $3), NEW ST_POINT($2, $3)) = 1`,
```
[the remark this answers is not in the document]
Comment (gdey) [reply provider/hana/util_internal_test.go:82]: I generated new values. Did not conform the values.

[the remark this answers is not in the document]
Comment (ARolek) [reply provider/hana/util_internal_test.go:82]: Ok, maybe check the old values to see how close they line up.

[the remark this answers is not in the document]
Comment (gdey) [reply provider/hana/util_internal_test.go:82]: I looked at it's all within `0.00000001` of each other. Some of the number are close to what was there, but the numbers after are different. I think this is again due to the drift in floating numbers.

**Comment by ARolek** [comment]
@gdey one nit on this PR, can you rename the commit titles? They're currently the same, but one commit is for vendoring the geom package, the other is for adopting it.

**Pull request merged**